### PR TITLE
Nathanassisnmentunittest

### PR DESCRIPTION
**Pull request by nncsumb**
Adds assignment controllers unit tests

### CODE REVIEW

**Comment by kukuharu** [comment]
Good use of mockbean, although I'm not sure where the GlobalExceptionHandler would go into v1.0 considering your structure was a bit different than everyone else's

File: src/main/java/com/exception/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(ResponseStatusException.class)
+    public ResponseEntity<Map<String, String>> handleResponseStatusException(ResponseStatusException ex) {
+        Map<String, String> errorResponse = new HashMap<>();
+        errorResponse.put("message", ex.getReason());
+        return new ResponseEntity<>(errorResponse, ex.getStatusCode());
+    }
+}
```
Comment (kukuharu) [inline src/main/java/com/exception/GlobalExceptionHandler.java:20]: good global exception handler

File: src/test/java/com/cst438/controller/AssignmentControllerUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+    // Helper method to convert objects to JSON string
+    private static String asJsonString(final Object obj) {
+        try {
+            return new ObjectMapper().writeValueAsString(obj);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+}
```
Comment (kukuharu) [inline src/test/java/com/cst438/controller/AssignmentControllerUnitTest.java:229]: Super well done, great use of mockbean

**Comment by Berkness32** [comment]
Everything looks good. We could probably merge whenever.

**Pull request merged**